android/app/src/main/java/com/microsoft/codepush/common/utils/PlatformUtils.java

```diff
@@ -2,14 +2,15 @@
 
 /**
  * Interface describing the methods that should be implemented in platform-specific instances of utils.
+ * It can be implemented via platform-specific singleton.
```

Comment:
Don't sure if we need this comment, is it some kind of todo?

Reply:
@sergey-akhalkov This is just a note on interface use case